templates/configure_ag.j2

```diff
@@ -219,7 +230,7 @@ BEGIN
   CREATE AVAILABILITY GROUP {{ mssql_ha_ag_name }}
     WITH (
       DB_FAILOVER = {{ __mssql_ha_db_failover }},
-      CLUSTER_TYPE = {{ mssql_ha_ag_cluster_type }}
+      CLUSTER_TYPE = {{ mssql_ha_ag_cluster_type }}  {{ is_contained }} {{ is_reuse_system_db }}
```

Comment:
Then, we can use this single variable here. I believe we do not need spaces between `{{ mssql_ha_ag_cluster_type }}` and `{{ is_contained }}` so that the value of CLUSTER_TYPE is comma-separated without extra spaces.

```suggestion
      CLUSTER_TYPE = {{ mssql_ha_ag_cluster_type }}{{ is_contained }}
```